source/manage-apps/configure/config/template-expansions.txt

```diff
@@ -86,21 +86,41 @@ Define a Template
             }
 
 
+   .. step:: Specify an Environment
+
+      Choose which environment to deploy and then update your app
```

Comment:
Do you deploy an environment, or deploy an app to an environment? I think the latter, but then the wording is a bit messier.